onboarding/src/Components/Navbar.js

```diff
@@ -27,50 +27,36 @@ function Navbar(props) {
     }
 
     return(
-        <nav className="top-bar navbar navbar-expand-lg navbar-dark bg-dark">
-            <a className="navbar-brand">Online Onboarding</a>
-            <button className="navbar-toggler" type="button" data-toggle="collapse"
-                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
-                    aria-label="Toggle navigation">
-                <span className="navbar-toggler-icon"/>
-            </button>
-            <div className="collapse navbar-collapse" id="navbarSupportedContent">
-                <ul className="navbar-nav d-flex">
-                    <li className="nav-item active">
-                        <a href={ path } className="nav-link">Home <span className="sr-only">(current)</span></a>
-                    </li>
-                    <li className="nav-item">
-                        <a className="nav-link">Link</a>
-                    </li>
-                    <li className="nav-item dropdown">
-                        <a className="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
-                           data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                            Dropdown
-                        </a>
-                        <div className="dropdown-menu" aria-labelledby="navbarDropdown">
-                            <a className="dropdown-item" href="#">Akcja</a>
-                            <a className="dropdown-item" href="#">Reakcja</a>
-                            <div className="dropdown-divider"/>
-                            <a className="dropdown-item" href="#">Lorem Ipsum</a>
-                        </div>
-                    </li>
-                </ul>
-                <div className="my-2 my-lg-0 col-6 p-0">
-                    <form className="form-inline">
-                        <input className="form-control mr-sm-2 col-9" type="search" placeholder="Szukaj" aria-label="Search"/>
-                        <button className="btn btn-outline-success my-2 my-sm-0 col-2 float-left" type="submit">Szukaj</button>
-                    </form>
-                </div>
-                {/*<div className="ml-4">
-                    <div className="form-inline my-2 my-lg-0">
-                        <p><em>Witaj #login</em></p>
-                    </div>
-                </div>*/}
+        // <nav className="top-bar navbar navbar-expand-lg navbar-dark">
+       <nav className="top-bar navbar p-0 flex-nowrap">
+            <div className="top-bar-brand pr-5" style={{height: "auto"}}>
+                <Link exact to="/" className="navbar-brand">Online Onboarding</Link>
```

Comment:
We may need to include here
`{{ pathname: "/", state: { packageId: packageId, loggedUser: loggedUser  } }}` as usual -
- we want the user to be kept on all components (the same for package if we want it).

Reply:
Done.